tests/interface_test.py

```diff
@@ -585,7 +585,7 @@ def check_for_basic_auth(self, request):
         self.assertEqual(request.headers["Authorization"], "Basic {}".format(b64))
 
     def assert_repos_equal(self, repo, expected):
-        self.assertEqual(repo.repo_id, expected.repo_id)
+        self.assertEqual(repo.id, expected.id)
```

Comment:
Could we add a `repo_id` property to `GogsRepo`? I'm fine with having an `id` property, but I'd like to keep `repo_id` to maintain backwards compatibility. (similarly for everything else renamed from `XYZ_id` to `id`)